saleor/dashboard/order/views.py

```diff
@@ -278,7 +277,7 @@ def ship_delivery_group(request, order_pk, group_pk):
 
 @staff_member_required
 @permission_required('order.edit_order')
-def cancel_delivery_group(request, order_pk, group_pk):
+def delivery_group_cancel(request, order_pk, group_pk):
```

Comment:
Why the weird naming?

Reply:
We were looking for a way to unify naming of those functions, but it does sound a bit unnatural, I agree.

Reply:
I'm not a fan. Neither Python nor Django use this notion to name their functions. Imagine using `object_get_or_404` or `request_process`.

Reply:
On the other hand, with this approach (naming functions as ```subject_action```) it's easier to search for functions related to a given model and to find a name for newly created ones. Although, I agree that it doesn't sound natural.

Reply:
If we wanted to follow `subject_action` we would come up with names like `delivery_group_variant_add` and `order_line_group_move_to`... 🐼

Reply:
In my opinion the original name was more natural, I'd recommend leaving it as is

Reply:
With Yoda, you convinced me. Change it back, I will. :laughing: